LandLord-core/src/main/java/biz/princeps/landlord/placeholderapi/LLExpansion.java

```diff
@@ -22,11 +22,6 @@ public String getIdentifier() {
         return "LandLord";
```

Comment:
Just confused here.
Doesn't this throw an error? According to the documentation the identifier need to be all lower case.
https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/PlaceholderExpansion#with-external-plugin

Reply:
Interestingly, the javadocs do not cover this <https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/src/main/java/me/clip/placeholderapi/expansion/PlaceholderExpansion.java#L44>. If it worked before, I think we can leave it for now.

Reply:
Indeed, it works perfectly currently. However, following the docs would make more sense.